tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -13,8 +14,32 @@
 
 namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests
 {
-    public sealed class OrganisationControllerTests 
+    public sealed class OrganisationControllerTests
     {
+        private Address address1 = new Address
```

Comment:
The address fields here should be marked as readonly as this will ensure the address data cannot be pointed to a new value. Also the convention for member variables is to prefix them with an underscore.